docs/1.0. Certificate Provisioning.md

```diff
@@ -192,9 +192,11 @@ The device manufacturer MUST make the public key of the CA available to the cust
 
 An EST Client SHOULD allow EST to be disabled, preventing the EST Client from being automatically provisioned with a TLS Certificate if required by the network's security policy. The default value SHOULD be EST **Enabled**.
```

Comment:
Could this line be removed as a result of this addition?

Reply:
I don't think so, as covers a different use case where you don't want to use EST at all, eg you want to manually manage the certificates and don't want it renew its cert using a rogue EST server on the network.
Happy to remove if we think this is not a valid use case?

Reply:
We will review and decide to remove or keep at next week's meeting.

Reply:
I agreed with James, to keep it in.